code/datums/gamemode/role/weeaboo.dm

```diff
@@ -0,0 +1,32 @@
+/datum/role/weeaboo
+	name = WEEABOO 
+	id = WEEABOO
```

Comment:
This line needs to be set to `ROLE_WEEABOO` for the polling and rule executing to actually work. However, this breaks the macro, and isn't how it's supposed to work if you look at all the other roles.

I having a hard time figuring out how to resolve this.